testData/typeAlias/unionTypeParams.d.ts

```diff
@@ -0,0 +1,14 @@
+type Values<V> = List<V>;
+type ListOrMultiMap<K,V> = List<V> | Map<K,Values<V>>;
```

Comment:
Another (very important) reason is that we can expand functions with union type in the signature (generate overloads).